lms/djangoapps/verify_student/views.py

```diff
@@ -918,7 +924,15 @@ def _compose_message_reverification_email(
                 unicode(location_id)
             )
         )
-        context["reverify_link"] = re_verification_link
+
+        reverify_link = "{site}{re_verification_link}".format(site=site, re_verification_link=re_verification_link)
+        if is_secure:
+            context["course_link"] = "https://{}".format(course_link)
+            context["reverify_link"] = "https://{}".format(reverify_link)
+        else:
+            context["course_link"] = "http://{}".format(course_link)
+            context["reverify_link"] = "http://{}".format(reverify_link)
```

Comment:
We seem to be doing a lot of URL manipulation here.  An alternative would be to pass the `request` object to this method, then use `request.build_absolute_uri(path)`.  This should automatically use the correct site name and protocol.